.circleci/config.yml

```diff
@@ -15,8 +15,8 @@ commands:
           keys:
             - v2-dependencies-{{ arch }}-{{ checksum "package-lock.json" }}
       - run:
-          name: install npm 6.9.0
-          command: sudo npm -g install npm@6.9.0
+          name: install latest npm
+          command: sudo npm -g install npm
```

Comment:
Why change this?

Reply:
because I'll change it everywhere? I think we should always use the latest npm.
since npm shows a nice banner when it's outdated, I think that users tend to do it.
also by using the latest npm, we might discover problems with new versions ealry.